subscriptions-api.md

```diff
@@ -373,6 +372,7 @@ Each subscription is represented by an object that has the following properties:
 
 - Constraints:
   - OPTIONAL for both subscription managers and subscribers to support
+  - If present, the array MUST have at least one filter expression
```

Comment:
I can't recall the exact topic, but didn't we have a concern where the diff between "empty" and "missing" was one that some implementations couldn't handle? I bring this up because I'm wondering if we really should ban a zero-length filter array?

Reply:
Maybe for this special case, we can have an empty one? I have no strong arguments about it.

Reply:
I'd prefer to allow for an empty array since someone will do it and I'm not sure why we would ban it. It seems unnecessarily strict.

Reply:
@duglin now it should be good to go, check the last commit